libs/sdk-ui-dashboard/src/model/store/meta/metaSelectors.ts

```diff
@@ -186,6 +204,15 @@ export const selectDashboardShareStatus = createSelector(selectDashboardDescript
     return state.shareStatus;
 });
 
+/**
+ * Selects dashboard share status.
+ *
+ * @alpha
+ */
+export const selectDashboardShareStatusPrivate = createSelector(selectDashboardShareStatus, (status) => {
```

Comment:
`selectIsDashboardPrivate` ? Also the description of the selector says, that it returns a share status, but it's not true.